doc/user/content/cli/_index.md

```diff
@@ -200,3 +202,61 @@ etc.), and then create a new node with those items.
 [gh-feature]: https://github.com/MaterializeInc/materialize/issues/new?labels=C-feature&template=feature.md
 [scv]: /sql/show-create-view
 [scs]: /sql/show-create-source
+
+### Persistence
+
+{{< warning >}}
+
+Persistence is a experimental feature at this time. Please don't use it
+with any data you are not comfortable losing. To use persistence you need to have
+both the `--persistence` and `--experimental` flags enabled.
+
+{{< /warning >}}
+
+Materialize lets you persist input data from Kafka sources to files on the
```

Comment:
One of my technical writing tricks is to shape things as "[What the user wants to accomplish], [how you do it in our product]." This increases the likelihood that the person reading the docs sees that we can help them achieve some  task. So I might suggest rewording this into something like:

> To avoid re-reading data from Kafka on restart, Materialize lets you persist input data from Kafka sources to files on the Materialize instance's local hard drive.

Reply:
done